### PR TITLE
adds basic navigation to all available screens via nav drawer

### PR DESCRIPTION
**Pull request by bamatson**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
             >
         </activity>
         <activity
-            android:name=".spending.MonthlySpendingActivity"
+            android:name=".spending.MonthlySpendingFragment"
```
Comment (matthew-carroll) [inline app/src/main/AndroidManifest.xml]: Looks like the wrong name here.

File: app/src/main/java/io/mattcarroll/androidtesting/home/HomeActivity.java [hunk 2]
```diff
@@ -122,24 +132,17 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
-    @SuppressWarnings("StatementWithEmptyBody")
     @Override
-    public boolean onNavigationItemSelected(MenuItem item) {
+    public boolean onNavigationItemSelected(@NonNull MenuItem item) {
```
Comment (matthew-carroll) [inline app/src/main/java/io/mattcarroll/androidtesting/home/HomeActivity.java]: I think this class also needs to highlight the initial selection upon creation or resume

File: app/src/main/res/layout/fragment_monthly_spending.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="io.mattcarroll.androidtesting.spending.MonthlySpendingActivity"
+    tools:context="io.mattcarroll.androidtesting.spending.MonthlySpendingFragment"
```
Comment (matthew-carroll) [inline app/src/main/res/layout/fragment_monthly_spending.xml]: "context" probably doesn't make sense here any more now that it's a Fragment

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (bamatson) [inline app/src/main/AndroidManifest.xml]: whoops

**Pull request merged**